rpc/wordvault/api.proto

```diff
@@ -112,6 +112,29 @@ message GetDailyLeaderboardResponse {
   repeated LeaderboardItem items = 1;
 }
 
+enum FsrsScheduler {
+    FSRS_SCHEDULER_NONE = 0;
+    FSRS_SCHEDULER_LONG_TERM = 1;
+    FSRS_SCHEDULER_SHORT_TERM = 2;
+}
+
+message FsrsParameters {
```

Comment:
Light wrapper ontop of the fsrs package -- no strong opinion on using this approach vs. just sending JSON and trusting the formatting. The one nicety here is that (I think) we get client-side codegen with types this way, whereas just JSON would not do that

However, this is a slight inconsistency with `cardRepr` right now, which does just send JSON

Reply:
That's ok. I wanted that one to eventually be typed a little stronger too, but this is good.